diff --git a/node_modules/electron-chrome-web-store/dist/cjs/browser/index.js b/node_modules/electron-chrome-web-store/dist/cjs/browser/index.js
index 4580b78..48a6788 100644
--- a/node_modules/electron-chrome-web-store/dist/cjs/browser/index.js
+++ b/node_modules/electron-chrome-web-store/dist/cjs/browser/index.js
@@ -977,8 +977,8 @@ async function loadAllExtensions(session, extensionsPath, options = {}) {
       }
       if (extension && extension.manifest.manifest_version === 3 && extension.manifest.background?.service_worker) {
         const scope = `chrome-extension://${extension.id}`;
-        await session.serviceWorkers.startWorkerForScope(scope).catch(() => {
-          console.error(`Failed to start worker for extension ${extension.id}`);
+        await session.serviceWorkers.startWorkerForScope(scope).catch((error) => {
+          console.error(`Failed to start worker for extension ${extension.id}:`, error);
         });
       }
     } catch (error) {
diff --git a/node_modules/electron-chrome-web-store/dist/esm/browser/index.mjs b/node_modules/electron-chrome-web-store/dist/esm/browser/index.mjs
index 85ba0e0..0024a7d 100644
--- a/node_modules/electron-chrome-web-store/dist/esm/browser/index.mjs
+++ b/node_modules/electron-chrome-web-store/dist/esm/browser/index.mjs
@@ -254,8 +254,8 @@ async function loadAllExtensions(session, extensionsPath, options = {}) {
       }
       if (extension && extension.manifest.manifest_version === 3 && extension.manifest.background?.service_worker) {
         const scope = `chrome-extension://${extension.id}`;
-        await session.serviceWorkers.startWorkerForScope(scope).catch(() => {
-          console.error(`Failed to start worker for extension ${extension.id}`);
+        await session.serviceWorkers.startWorkerForScope(scope).catch((error) => {
+          console.error(`Failed to start worker for extension ${extension.id}:`, error);
         });
       }
     } catch (error) {
